hack/unwanted-dependencies.json

```diff
@@ -204,6 +206,23 @@
         "sigs.k8s.io/kustomize/kustomize/v5",
         "sigs.k8s.io/kustomize/kyaml"
       ],
+      "github.com/modern-go/concurrent": [
+        "github.com/json-iterator/go",
+        "github.com/prometheus/client_golang",
+        "go.etcd.io/etcd/client/v2",
+        "go.etcd.io/etcd/server/v3",
+        "go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful",
+        "k8s.io/kube-openapi",
+        "sigs.k8s.io/structured-merge-diff/v4"
+      ],
+      "github.com/modern-go/reflect2": [
+        "github.com/json-iterator/go",
+        "github.com/prometheus/client_golang",
+        "go.etcd.io/etcd/client/v2",
+        "go.etcd.io/etcd/server/v3",
+        "go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful",
+        "k8s.io/kube-openapi"
```

Comment:
via structured-merge-diff → json-iterator, removal tracked in https://github.com/kubernetes-sigs/structured-merge-diff/issues/202